Don't attempt to add the module with a LoadModule line to the
httpd.conf file. This won't work in the buildroot and also is
not how modules are loaded in Apache in (open)SUSE.

Index: php-7.4.13/sapi/apache2handler/config.m4
===================================================================
--- php-7.4.13.orig/sapi/apache2handler/config.m4	2020-11-24 11:03:34.000000000 +0100
+++ php-7.4.13/sapi/apache2handler/config.m4	2020-12-04 17:45:39.679663555 +0100
@@ -64,18 +64,9 @@ if test "$PHP_APXS2" != "no"; then
   fi
 
   APXS_LIBEXECDIR='$(INSTALL_ROOT)'`$APXS -q LIBEXECDIR`
-  if test -z `$APXS -q SYSCONFDIR`; then
-    INSTALL_IT="\$(mkinstalldirs) '$APXS_LIBEXECDIR' && \
+  INSTALL_IT="\$(mkinstalldirs) '$APXS_LIBEXECDIR' && \
                  $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
                        -i -n php7"
-  else
-    APXS_SYSCONFDIR='$(INSTALL_ROOT)'`$APXS -q SYSCONFDIR`
-    INSTALL_IT="\$(mkinstalldirs) '$APXS_LIBEXECDIR' && \
-                \$(mkinstalldirs) '$APXS_SYSCONFDIR' && \
-                 $APXS -S LIBEXECDIR='$APXS_LIBEXECDIR' \
-                       -S SYSCONFDIR='$APXS_SYSCONFDIR' \
-                       -i -a -n php7"
-  fi
 
   case $host_alias in
   *aix*)
